pinot-broker/pom.xml

```diff
@@ -75,8 +75,8 @@
       <artifactId>jersey-container-grizzly2-http</artifactId>
     </dependency>
     <dependency>
-      <groupId>org.glassfish.jersey.containers</groupId>
-      <artifactId>jersey-container-grizzly2-servlet</artifactId>
+      <groupId>org.glassfish.jersey.media</groupId>
+      <artifactId>jersey-media-json-jackson</artifactId>
```

Comment:
r u sure you need this? If you can try without this that may be better.

Reply:
I will remove this.

Reply:
pls make sure to remove from parent pom also